bsp/boards/board.h

```diff
@@ -22,16 +22,50 @@ typedef enum {
    KICK_SCHEDULER        = 1,
 } kick_scheduler_t;
 
+typedef enum {
+   TIME_MS,
+   TIME_TICS,
+} time_type_t;
+
 //=========================== typedef =========================================
+typedef struct {
+    uint16_t PORT_SLOTDURATION;

```

Comment:
- no need to keep the prefix `PORT`.
- don't use all caps if now a variable
- why is there both `PORT_SLOTDURATION` and `PORT_TsSlotDuration`?

Suggested names:
- `SlotDuration`
- `slotDuration`
- `maxTxDataPrepare`
etc.